src/newbank/server/database/AccountDB.java

```diff
@@ -0,0 +1,47 @@
+package newbank.server.database;
+
+import newbank.server.model.Account;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AccountDB {
```

Comment:
I do not think this class is necessary at all. I thought we should always associate an account with a user. This AccountDB allows in having orphaned accounts. Do we want to have this?

Reply:
In Customers I am tracking only this account number
private ArrayList accountNumbers;

This will be the key to the account. This is done so because transfer of amount is always done between account to account. We should be able to identity account by the account number itself. Maybe when we implemented something later with account number this will come handy. Orphaned accounts wont be created as the account addition is always done through Customer class. addAccount(Account account)

Reply:
I would agree that the accountDB would be helpful. While there is no requirement in the protocol to transfer between different accounts using the account number, this seems a sensible thing to do.
The interesting question is how we ensure that we do not store redunant data, since in the moment we also have copies of the accounts in the customer.java.